tests/c_api_tests/test_c_api_model.cpp

```diff
@@ -46,7 +46,7 @@ TEST_CASE("C API Model") {
     PGM_Options* opt = unique_options.get();
 
     // input data
-    ConstDatasetPtr const unique_input_dataset{PGM_create_dataset_const(hl, "input", false, 1)};
+    ConstDatasetPtr const unique_input_dataset{PGM_create_dataset_const(hl, "input", 0, 1)};
```

Comment:
Why is true/false not correct?

Reply:
implicit casts are often dangerous, especially when the types don't map to eachother bijectionally. In this case, there's a many-to-one relation (surjection) from all nonzero integer values to `true` (including negative values). To avoid confusion and to make it explicit, it's recommended to either use `static_cast` every single time, OR to use integer values.

Although C++ guarantees that `true == 1`, implicit casting may e.g. result in `5 -> true -> 1`, which may result in unexpected behaviour